### DISCUSSION TITLE
Auto delete msg for everyone... after X seconds

### DISCUSSION THREAD

**Discussion by BroodPitt** [vasani-arpit/WBOT#252] [Ideas]
Hi!,

Is it possible to add a feature that the bot deletes its message after X sec (for everyone)

gr
BroodPitt

**Comment by vasani-arpit** [comment]
Yes. It is possible and can be done but what is the use case? In which case you might need it?

**Reply by BroodPitt** [reply]
I use this for our IT team, so we can use like

!laps HOSTNAME
!username Name or username (to get info about the user)

We get the password of the machine (local admin pass, with LAPS)

but... i want that the message (password) is deleted after 10seconds :D


edit: and it would be nice that you can set it in the JSON/Webhook and per action like
!laps == 10seconds then delete
!username == 25seconds then delete
etc...


We only use the webhook (php, to connect to LDAP and a Database :))

**Reply by vasani-arpit** [reply]
Yes. Nice use case. wouldn't that be great if we count the seconds after user reads the message. direct flat X amount seconds doesn't make sense because user might be offline.

**Reply by BroodPitt** [reply]
after reading would be great!, but does that work if the user has disabled the blue check/read thing? :O

if someone types the command.. i hope they read it immediately :D

**Reply by vasani-arpit** [reply]
Yes. read receipt thing is hard to implement as well. Better to go with time based and send another message after deleting mentioning that time window to see this message has passed to message has been deleted.

**Comment by BroodPitt** [comment]
Any news? :)

**Reply by vasani-arpit** [reply]
I am working on multi-device support of web version. I think it won't productive if I add this feature in current version of the web because it will be replaced by new multi-device whatsapp web.
